common/build.gradle

```diff
@@ -9,6 +9,10 @@ dependencies {
         exclude module: 'slf4j-api'
     }
 
+    implementation 'org.mongodb:mongodb-driver-sync:4.6.0'
```

Comment:
Let's compileOnly on these and load at runtime by placing them in the libraries section of  `plugin.yml`
Reason being, we can't ship jars larger than 2MB as that's Spigot's upload limit.
Prior to this, the jar size is 1,950KB -- 10KB under the limit. It's very important that we stay under that limit.